deploy/helm/distributed-compute-operator/templates/deployment.yaml

```diff
@@ -69,10 +69,10 @@ spec:
             - --istio-enabled
             {{- end }}
             {{- with .Values.mpi.initImage }}
-            - --mpi-init-image={{ . }}
+            - --mpi-init-image="{{ .registry }}/{{ .repository }}:{{ .tag }}"
```

Comment:
I believe you can do `{{ include "common.imagename" .Values.mpi.initImage }}`

Reply:
This is the point of using the split up OCI format

Reply:
should it be "common.images.image"? https://artifacthub.io/packages/helm/bitnami/common#images